util/zenfs.cc

```diff
@@ -69,21 +70,56 @@ Status zenfs_mount(std::unique_ptr<ZonedBlockDevice> &zbd,
   return s;
 }
 
+// Create or check pre-existing aux directory and fail if it is
+// inaccessible by current user and if it has previous data
+int create_aux_path() {
+  struct dirent *dent;
+  DIR *aux_dir;
+  unsigned int nfiles = 0;
```

Comment:
almost always `std::size_t` for counters
```suggestion
  std::size_t nfiles = 0;
```

Reply:
Ok